lib/bloc/account/account_model.dart

```diff
@@ -230,6 +230,9 @@ class PaymentInfo {
     if (_paymentResponse.invoiceMemo.description.startsWith("Bitrefill")) {
       return "Bitrefill";
     }
+    if (_paymentResponse.invoiceMemo.description.startsWith("Fastbitcoins")) {
```

Comment:
No need for this, we are using our invoice here and we put the description as we want it on the creation fo the invoice.

Reply:
This is for title of payment list.

Reply:
But we don't need it because the description is ok. "Fastbitcoins.com Voucher"

Reply:
I see... Well, then I'm changing maxLines of PaymentItem widget's title to 1. It was splitting into multiple lines on my device.